explorer/lib/explorer/models/proofs.ex

```diff
@@ -42,4 +42,29 @@ defmodule Proofs do
     end
   end
 
+  def get_batch_from_proof(proof_hash_hex) do
```

Comment:
if it returns N, function name should be plural
```suggestion
  def get_batches_containing_proof(proof_hash_hex) do
```